src/baal/utils/metrics.py

```diff
@@ -98,9 +102,24 @@ def __init__(self, n_bins=10, **kwargs):
         super().__init__(average=False)
 
     def update(self, output=None, target=None):
+        """
+        Updating the true positive (tp) and number of samples in each bin.
+
+        Args:
+            output (tensor): logits or predictions of model
+            target (tensor): labels
+        """
+        output = output.detach().cpu().numpy()
+        target = target.detach().cpu().numpy()
+        output = to_prob(output)
```

Comment:
is there a reason why softmax is not called explicitly and replaced by to_prob instead?

Reply:
It looks like in the wrapper case, to_prop was extracted for clarity. Then it's best to use it consistently so that if it changes in the to_prob method, it'll take effect everywhere. In this case we only want to take the softmax if there are values outside [0,1] or if they don't sum to 1.